sdk/scheduler/src/test/java/com/mesosphere/sdk/dcos/CapabilitiesTest.java

```diff
@@ -96,6 +96,34 @@ public void test_110dev() throws IOException {
     }
 
     @Test
+    public void test_1110() throws IOException {
+        Capabilities capabilities = testCapabilities("1.11.0");
+        Assert.assertTrue(capabilities.supportsNamedVips());
+        Assert.assertTrue(capabilities.supportsGpuResource());
+        Assert.assertTrue(capabilities.supportsRLimits());
+        // Secrets
+        Assert.assertTrue(capabilities.supportsEnvBasedSecretsDirectiveLabel());
+        Assert.assertTrue(capabilities.supportsEnvBasedSecretsProtobuf());
+        Assert.assertTrue(capabilities.supportsFileBasedSecrets());
+
+        Assert.assertTrue(capabilities.supportsStrictModeV1API());
+    }
+
+    @Test
+    public void test_111dev() throws IOException {
+        Capabilities capabilities = testCapabilities("1.11-dev");
+        Assert.assertTrue(capabilities.supportsNamedVips());
+        Assert.assertTrue(capabilities.supportsGpuResource());
+        Assert.assertTrue(capabilities.supportsRLimits());
+        // Secrets
+        Assert.assertTrue(capabilities.supportsEnvBasedSecretsDirectiveLabel());
+        Assert.assertTrue(capabilities.supportsEnvBasedSecretsProtobuf());
+        Assert.assertTrue(capabilities.supportsFileBasedSecrets());
+
+        Assert.assertTrue(capabilities.supportsStrictModeV1API());
```

Comment:
We don't seem to check that other versions (at least one) do *not* support this API?

Reply:
Indeed we do not. @mrbrowning can you throw a test under 1.10?